benchmarks/benchmarks/bench_avx.py

```diff
@@ -16,11 +16,11 @@
 dtype  = ['f', 'd']
 
 class AVX_UFunc(Benchmark):
-    params = [avx_ufuncs, stride, dtype]
-    param_names = ['avx_based_ufunc', 'stride', 'dtype']
+    params = [avx_ufuncs, dtype, stride]
+    param_names = ['avx_based_ufunc', 'dtype', 'stride']
     timeout = 10
 
-    def setup(self, ufuncname, stride, dtype):
+    def setup(self, ufuncname, dtype, stride):
```

Comment:
Note that swapping the order of these arguments breaks comparison across benchmarks. I don't think this minor reordering justifies that.

Reply:
Sure, I will switch it back.